docs/developer/independent-container-updates.md

```diff
@@ -0,0 +1,83 @@
+# Independent Container Updates
+
+Since version 0.10.0, Dangerzone is able to ship container images independently
+from releases of the software.
+
+One of the main benefits of doing so is to shorten the time neede to distribute the security fixes for the containers. Being the place where the actual conversion of documents happen, it's a way to keep dangerzone users secure.
+
+If you are a dangerzone user, this all happens behind the curtain, and you should not have to know anything about that to enjoy these "in-app" updates. If you are using dangerzone in an air-gapped environment, check the sections below.
```

Comment:
That makes me think that we should refer to the container image as "the sandbox" everywhere in the app. Wording changes are needed.